src/lib/components/debug/loadChart.svelte

```
@@ -8,36 +9,100 @@
+                    <span>{formatDuration(chart.durationMs)}</span>
+                </div>
+            </div>
+        </div>
+        {#if reference || legend.length > 0}
+            <ul class="text-muted-foreground mt-1.5 space-y-0.5 text-[11px] leading-snug">
+                {#if reference}
+                    <li>Пунктир — {reference.label}</li>
+                {/if}
+                {#each legend as item}
+                    <li>{item}</li>
+                {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)